cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,31 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
+                user_partitions = item.user_partitions
+                split_partitions = []
+                if user_partitions:
+                    for user_partition in user_partitions:
+                        if user_partition.scheme.name == RANDOM_SCHEME:
+                            split_partitions.append(user_partition)
+                for vertical in item.get_children():
```

Comment:
Just to clarify: the vertical convention is something that we did in Studio to make it simple to manage a collection of children shown to a particular group. The underlying platform doesn't make such an assumption.